egs2/ksc/asr1/run.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+train_set="train"
+valid_set="dev"
+test_sets="test dev"
+
+asr_config=conf/train.yaml
+inference_config=conf/decode.yaml
+
+#if stage 10 fails, try setting the '--nj 1'.
```

Comment:
```suggestion
```
I think it is mostly due to the limited memory in local machine, so it would not be a general issue I believe